flycheck.el

```diff
@@ -9350,6 +9350,13 @@ See URL `http://pear.php.net/package/PHP_CodeSniffer/'."
             ;; been configured with show_progress enabled
             "-q"
             (option "--standard=" flycheck-phpcs-standard concat)
+            ;; Some files are not detected correctly
+            ;; so it is necessary to pass the extension.
+            (eval
+             (-when-let* ((fname buffer-file-name)
+                          (ext (file-name-extension fname))) 
```

Comment:
Almost there, but now you can actually use `ext` instead of `(file-name-extension buffer-file-name)` in the `concat`

Reply:
Ok, really thank you for your patience